src/test/java/seedu/address/logic/commands/ConfirmCommandTest.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.person.Person;
+import seedu.address.testutil.EditPersonDescriptorBuilder;
+import seedu.address.testutil.PersonBuilder;
+
+public class ConfirmCommandTest {
+    @Test
+    public void equals() {
+        DeleteCommand firstDeleteCommand = new DeleteCommand(INDEX_FIRST_PERSON);
+        DeleteCommand secondDeleteCommand = new DeleteCommand(INDEX_SECOND_PERSON);
+
+        Person editedPerson = new PersonBuilder().build();
+        EditCommand.EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(editedPerson).build();
+        EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
+
+        ClearCommand clearCommand = new ClearCommand();
+
+        ConfirmCommand confirmationDeleteCommand = new ConfirmationCommand(firstDeleteCommand);
+        ConfirmCommand confirmationEditCommand = new ConfirmationCommand(editCommand);
+        ConfirmCommand confirmationClearCommand = new ConfirmationCommand(clearCommand);
+
+        ConfirmCommand confirmationAcceptDeleteCommand = new ConfirmationAcceptCommand(firstDeleteCommand);
+        ConfirmCommand confirmationAcceptEditCommand = new ConfirmationAcceptCommand(editCommand);
+        ConfirmCommand confirmationAcceptClearCommand = new ConfirmationAcceptCommand(clearCommand);
+
+        ConfirmCommand confirmationRejectDeleteCommand = new ConfirmationRejectCommand(firstDeleteCommand);
+        ConfirmCommand confirmationRejectEditCommand = new ConfirmationRejectCommand(editCommand);
+        ConfirmCommand confirmationRejectClearCommand = new ConfirmationRejectCommand(clearCommand);
+
+        assertTrue(confirmationDeleteCommand.equals(confirmationDeleteCommand));
+        assertTrue(confirmationEditCommand.equals(confirmationEditCommand));
+        assertTrue(confirmationClearCommand.equals(confirmationClearCommand));
+
+        assertFalse(confirmationDeleteCommand.equals(new ConfirmationCommand(secondDeleteCommand)));
+
+        assertFalse(confirmationDeleteCommand.equals(confirmationClearCommand));
+        assertFalse(confirmationDeleteCommand.equals(confirmationEditCommand));
+        assertFalse(confirmationEditCommand.equals(confirmationClearCommand));
```

Comment:
Perhaps comments could be added to explain what you're testing for each statement (+ expected result)?

Reply:
Thank you for the comment, I have resolved it!